.github/workflows/idefix-ci-doc.yml

```diff
@@ -0,0 +1,26 @@
+name: Idefix CIs
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+    paths-ignore:
+      - '.github/ISSUE_TEMPLATE/*'
+
+
+jobs:
+  ReadTheDocs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: install dependencies
+        run: sudo apt-get install -y doxygen
+      - name: install doxygen
```

Comment:
Looks like the name of this step got swapped with the previous one

Reply:
fixed